### ISSUE TITLE
Dynamic Global Translation Variables

### ISSUE THREAD

**Issue by pedami** [thedevdojo/voyager#1526]
- Laravel Version: last
- Voyager Version: last
- PHP Version: 7
- Database Driver & Version: mysql last

### Description:
Hello, I have a query to a table of translations that I created in Voyager with the fields id, key, translation, where translation is translatable
I have inserted this small code in the boot of AppServiceProvider.php to have the variables that are created in the registers to have a fast translation system, but only works in the default language
Any idea why the getTranslatedAttribute method does not just work?
 

```
namespace App\Providers;

use Illuminate\Http\Request;
use App\Traduccione;
use Illuminate\Support\ServiceProvider;
use Illuminate\Support\Facades\Schema;
use View;

class AppServiceProvider extends ServiceProvider
{
    /**
     * Bootstrap any application services.
     *
     * @return void
     */
    public function boot()
    {
		Schema::defaultStringLength(191);

        $traduccion = Traduccione::all();
        foreach ($traduccion as $traduc) {
            $TraVar_DW = $traduc->clave;
            ${$TraVar_DW} = $traduc->getTranslatedAttribute('traduccion');
            // dd($inicio);
        View::share(''.$TraVar_DW.'', ${$TraVar_DW});
        }
    }

    /**
     * Register any application services.
     *
     * @return void
     */
    public function register()
    {
/*
		$this->app->bind('path.public', function() {
			return base_path().'/httpdocs';
		});
*/
    }
}
```

**Comment by lancepioch** [comment]
Labels: locale, possible bug

**Comment by fletch3555** [comment]
This should be fixed in dev-master and will be included in 1.0.1
